tests/integration_tests.rs

```diff
@@ -756,6 +892,95 @@ fn config_read_arguments_from_file() {
         .stdout(predicate::eq("dummy-pager-from-config\n").normalize());
 }
 
+// ignore this test for now as cargo cache --clear only targets the default projects dir
+#[cfg(unix)]
+#[test]
+#[ignore]
+fn cache_clear() {
+    let src_dir = "cache_source";
+    let tmp_dir = tempdir().expect("can create temporary directory");
+    let themes_filename = "themes.bin";
+    let syntaxes_filename = "syntaxes.bin";
+    let metadata_filename = "metadata.yaml";
+    [themes_filename, syntaxes_filename, metadata_filename]
+        .iter()
+        .map(|filename| {
+            let fp = tmp_dir.path().join(filename);
+            let mut file = File::create(fp).expect("can create temporary file");
+            writeln!(file, "dummy content").expect("can write to file");
+        })
+        .count();
+
+    // Clear the targeted cache
+    bat_with_config()
+        .current_dir(Path::new(EXAMPLES_DIR).join(src_dir))
+        .env("BAT_CONFIG_PATH", "bat.conf")
+        .env("BAT_THEME", "1337")
```

Comment:
Is this relevant here?

Reply:
These two lines are to ensure that running `bat cache --clear` with an BAT config and BAT env variable set does not break the command. Since I've added code that inserts arguments before the command-line args, we want to ensure that it does not insert them before the `cache` subcommand.

Reply:
Ah, now I understand. Could we maybe add your explanation here as a code comment?

Reply:
Sure. Should I comment each of the tests where this happens then?

Reply:
Maybe yes. It's not obvious from the test names (the tests are useful even without checking this env. var behavior, so I would also keep the test name generic).

Reply:
ah ok, it's only two places - I'll comment both. The other tests have names that are more specific to testing the env. var behavior, so they are fine as is.